src/main/java/seedu/address/logic/parser/DeleteCommandParsers/DeleteMentorCommandParser.java

```diff
@@ -11,17 +16,17 @@
 
     /**
      * Parses the given {@code String} of arguments in the context of the DeleteCommand
-     * and returns a DeleteCommand object for execution.
+     * and returns a DeleteMentorCommand object for execution.
      * @throws ParseException if the user input does not conform the expected format
      */
-    public DeleteCommand parse(String args) throws ParseException {
-
-        /**
-         * The below is just placeholder code until the Mentor class
-         * is finalised after which we will replace with actual code.
-         */
-
-        return null;
+    public DeleteMentorCommand parse(String args) throws ParseException {
+        try {
+            Id id = AlfredParserUtil.parseIndex(args, PrefixType.M);
+            return new DeleteMentorCommand(id);
+        } catch (ParseException pe) {
+            throw new ParseException(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteMentorCommand.MESSAGE_USAGE), pe);
```

Comment:
@john0227 
Could you have a look at the AB3 code and make sure you "MESSAGE_USAGE" String so that I can use it in the parser and use it as a warning in case the user inputs an incorrect format for the delete command.